config/main.py

```diff
@@ -1221,16 +1224,18 @@ def replace(ctx, target_file_path, format, dry_run, verbose):
 @config.command()
 @click.argument('checkpoint-name', type=str, required=True)
 @click.option('-d', '--dry-run', is_flag=True, default=False, help='test out the command without affecting config state')
+@click.option('-n', '--non-strict', is_flag=True, default=False, help='ignore validation for changes to tables not YANG covered', hidden=True)
+@click.option('-i', '--ignore-more', multiple=True, help='used with -n option to ignore validation for more config. Requires JsonPointer', hidden=True)
```

Comment:
>'--ignore-more' [](http://example.com/codeflow?start=20&length=15)

`more` is not meaningful.
How about `--ignore-yang-outdated` ? `outdated` is just an example. I am not sure what is the true scope of the errors you are trying to ignore.

Reply:
the `-n` activates the non-strict mode which by default ignores tables without yang from validation.

In the non-strict also we can ignore more configs using the `-i` flag. I can name `ignore-more-from-yang-validation`

Reply:
renamed to `ignore-more-from-yang-validation`